File: Solutions/Easy/richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) 
    {
        int max_value = -1;
        for (auto a : accounts)
        {
            vector <int> val_vec = a;
            int total_value = 0;
            for (auto v : val_vec) 
            {
                total_value += v;
            }
            if (total_value > max_value)
            {
                max_value = total_value;
            }
        }
        return max_value;
    }
};

File: Java/easy/RichestCustomerWealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int customerWealth=0,maxWealth = 0;
        
        for ( int i=0; i<accounts.length; i++){
            for(int j=0; j<accounts[0].length; j++){
                customerWealth += accounts[i][j];
                if(customerWealth > maxWealth)
                    maxWealth = customerWealth;
            }
            customerWealth = 0;
        }
        return maxWealth;
    }
}